doc/telemetry/spire_grafana_dashboard.json

```diff
@@ -0,0 +1,3338 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": {
+            "type": "datasource",
+            "uid": "grafana"
+          },
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "target": {
+            "limit": 100,
+            "matchAny": false,
+            "tags": [],
+            "type": "dashboard"
+          },
+          "type": "dashboard"
+        }
+      ]
+    },
+    "editable": true,
+    "fiscalYearStartMonth": 0,
+    "graphTooltip": 0,
+    "id": 95,
+    "links": [],
+    "liveNow": true,
+    "panels": [
+      {
+        "collapsed": true,
+        "datasource": {
+          "type": "datasource",
+          "uid": "grafana"
+        },
+        "gridPos": {
+          "h": 1,
+          "w": 24,
+          "x": 0,
+          "y": 0
+        },
+        "id": 23,
+        "panels": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "uid_prometheus_grafana_datasource"
```

Comment:
I think that we can avoid having a static uid defined for the datasource if this would be defined as an input parameter and then refer to it as something like `${DS_UID_PROMETHEUS_GRAFANA_DATASOURCE}`. 
I think that this could be easily achieved if the "[export for sharing externally](https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/#export-a-dashboard-as-json)" option is selected when you have this dashboard exported.
That way, I think that this file would be easier to consume, as you don't need to have defined a data source with that exact same uid, but rather configure any data source and select it at the moment of importing.